examples_tests/13.BRDF_Explorer/BRDFExplorerApp.cpp

```diff
@@ -398,18 +413,18 @@ R"(
         const float LplusV_lenSq = 2.0 + 2.0*LdotV;
```

Comment:
are you going to use those identities?

Reply:
they were in use until i needed TdotH or BdotH (so now i need H and not only N,V dots with H)

Reply:
any vector, lets call it `K` can have `KdotH` calculated from identities

dot(K,H) = dot(K,L+V)/length(L+V) = (dot(K,L)+dot(K,V))*LplusV_rcpLen = (KdotL+KdotV)*LplusV_rcpLen 

You actually happen to have all TdotV,TdotL,BdotB,BdotL needed